src/models/resourcekindhandler.ts

```diff
@@ -126,6 +126,10 @@ interface ResourceKindHandler {
   }[];
 
   validationSchemaPrefix?: string;
+
+  formSchema?: any;
```

Comment:
there is already a formEditorOptions property in ResourceKindHandler  (see line 103) - perhaps you can use that property instead of adding these?